### ISSUE TITLE
Game minimized when opening quick menu - ROG ALLY

### ISSUE THREAD

**Issue by gogomimigo** [Valkirie/HandheldCompanion#670]
Hello. I installed HC and closed AC before starting HC. I set a hotkey to open the quick menu but whenever I'm in game and press the hotkey, the game will minimized and it will distort the whole desktop. The quick menu pops up but it kinda messes up the whole experience. Is there a workaround for this?

**Comment by trippyone** [comment]
@CasperH2O duplicate of #630

**Comment by CasperH2O** [comment]
@gogomimigo , the work around for this is to set games to windowed maximized or borderless fullscreen.

Closing because of duplicate.
